README.md

```diff
@@ -117,7 +117,7 @@ A list of some interesting expansions for the project:
 
 + ~Sort by (pixel) colour~
```

Comment:
Add `sort` to params in "How can I use it?" section

Reply:
oops, thanks. This is now fixed.